src/coreclr/jit/morph.cpp

```diff
@@ -10869,7 +10869,71 @@ GenTree* Compiler::fgOptimizeHWIntrinsic(GenTreeHWIntrinsic* node)
             INDEBUG(node->gtDebugFlags |= GTF_DEBUG_NODE_MORPHED);
             return node;
         }
+#if defined(TARGET_XARCH)
+        case NI_SSE_And:
+        case NI_SSE2_And:
+        case NI_AVX_And:
+        case NI_AVX2_And:
```

Comment:
What about Vector128/256_And and AdvSimd ?

Reply:
`Vector64/128/256_And` don't exist outside of `import` at the moment so they don't need to be handled.

`AdvSimd` should be since we want parity between xarch and arm.